setup.py

```diff
@@ -54,7 +54,8 @@
     if package == MODULE_DIR or package.startswith(MODULE_DIR + ".")
 ]
 install_requires = [
-    "urllib3>=1.26.18, <2",
+    'urllib3>=1.26.18,<1.27 ; python_version < "3.10"',
+    'urllib3>=1.26.18,!=2.2.0,<3 ; python_version >= "3.10"',
```

Comment:
Why the specific exclusion of 2.2.0?

Reply:
This is mirrors what botocore did. 2.2.0 has some breaking changes.

https://github.com/boto/botocore/pull/3141

https://github.com/boto/botocore/issues/3138#issuecomment-1986927748